Neos.Flow.Log/Classes/PsrLogger.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace Neos\Flow\Log;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Log\Backend\BackendInterface;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LoggerTrait;
+use Psr\Log\LogLevel;
+
+/**
+ *
+ */
+class PsrLogger implements LoggerInterface
```

Comment:
Hey @kitsunet, wouldn't it possible to make this PSR compatible logger the default one. And if so could we remove the PSR in the naming?